m4/with_libical.m4

```diff
@@ -72,12 +72,11 @@ AC_DEFUN([PBS_AC_WITH_LIBICAL],
   AC_SUBST(libical_inc)
   AC_SUBST(libical_lib)
   AC_DEFINE([LIBICAL], [], [Defined when libical is available])
-  AS_IF([test "$libical_dir" = "/usr"],
-    [PKG_CHECK_MODULES([libical_api2],
-      [libical >= 2],
-      [AC_DEFINE([LIBICAL_API2], [], [Defined when libical version >= 2])],
-      [echo "libical version 2 is not available"]
-    )]
+  export PKG_CONFIG_PATH=["${libical_dir}/lib/pkgconfig/"]
```

Comment:
Setting this variable will restrict to look into only this path for both the case like 1) --with_libical and 2) system installed libraries. 

It would be better to make this export for if the case where "--with-libical" is provided.